transport/transport_test.go

```diff
@@ -2156,3 +2166,72 @@ func TestReadGivesSameErrorAfterAnyErrorOccurs(t *testing.T) {
 		}
 	}
 }
+
+func TestPingPong1B(t *testing.T) {
+	runPingPongTest(t, 1)
+}
+
+func TestPingPong1KB(t *testing.T) {
+	runPingPongTest(t, 1024)
+}
+
+func TestPingPong64KB(t *testing.T) {
+	runPingPongTest(t, 65536)
+}
+
+func TestPingPong1MB(t *testing.T) {
+	runPingPongTest(t, 1048576)
+}
+
+func runPingPongTest(t *testing.T, msgSize int) {
```

Comment:
Can you add a comment to say what this test is supposed to catch that our other tests don't?